int32 nb_sec
float64 latitude
float64 longitude
float64 vitesse_nd
float64 heading